src/utilities/core-utils.ts

```diff
@@ -56,7 +107,12 @@ const _numericEnumToPojo = (enumObject: any): {} => {
     return pojo;
 };
 
-const _objectToArray = (object: any): any[] => {
+const memoize = <T extends (...args: any[]) => any>(
```

Comment:
Nitpick - alphabetization

Reply:
Yeah, I tried removing the comments I copied over and it wouldn't show anymore even with the direct export. I think it doesn't hurt to just copy over so I just do them all like that